src/adapters/curve/products/pool/curvePoolAdapter.ts

```diff
@@ -161,6 +167,10 @@ export class CurvePoolAdapter
       this.provider,
     )
 
+    if (this.chainId == 1 && blockNumber && blockNumber < 15732062) {
```

Comment:
```suggestion
    if (this.chainId == Chain.Ethereum && blockNumber && blockNumber < 15732062) {
```
Best to use the constant.